samples/bluetooth/direct_test_mode/src/dtm.h

```diff
@@ -156,11 +170,57 @@ struct dtm_tx_power {
 	bool max;
 };
 
+/** @brief DYM Packet status for IQ Sample report. */
+enum dtm_packet_status {
+	/** Packet received with proper CRC */
+	DTM_PACKET_STATUS_CRC_OK,
+
+	/** Packet received with invalid CRC.
+	 *  The Length and CTEInfo was used to calculate sampling points.
+	 */
+	DTM_PACKET_STATUS_CRC_ERR_TIME,
+
+	/** Packet received with invalid CRC.
+	 *  The sampling points were calculated in another way.
+	 */
+	DTM_PACKET_STATUS_CRC_ERR_OTHER,
+
+	/** Packet received with invalid CRC.
+	 *  Insufficient resources to sample.
+	 */
+	DTM_PACKET_STATUS_CRC_ERR_INSUFFICIENT
+};
+
+/** @brief IQ sample format. */
+struct dtm_iq_sample {
+	int16_t i;
+	int16_t q;
+};
+
+/** @brief DTM IQ sampling data with additional information. */
+struct dtm_iq_data {
+	uint8_t channel;
+	int16_t rssi;
+	uint8_t rssi_ant;
+	enum dtm_cte_type type;
+	enum dtm_cte_slot_duration slot;
+	enum dtm_packet_status status;
+	uint8_t sample_cnt;
+	struct dtm_iq_sample *samples;
+};
+
+typedef void (*dtm_iq_report_callback_t)(struct dtm_iq_data *data);
+
 /** @brief Function for initializing DTM module.
```

Comment:
We usually have the function brief descriptions in imperative format, so suggesting:
"Initialize the DTM module."